Index: docker-py/setup.py
===================================================================
--- docker-py.orig/setup.py
+++ docker-py/setup.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 from __future__ import print_function
 
 import codecs
@@ -11,10 +11,9 @@ from setuptools import setup, find_packa
 
 if 'docker-py' in [x.project_name for x in pip.get_installed_distributions()]:
     print(
-        'ERROR: "docker-py" needs to be uninstalled before installing this'
+        'WARNING: "docker-py" needs to be uninstalled before installing this'
         ' package:\npip uninstall docker-py', file=sys.stderr
     )
-    sys.exit(1)
 
 ROOT_DIR = os.path.dirname(__file__)
 SOURCE_DIR = os.path.join(ROOT_DIR)
